app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t "base_title"
+    if page_title.empty?
```

Comment:
Dùng toán tử 3 ngôi nhé
page_title.empty? ? base_title : "#{page_title}  | #{base_title}"